### PR TITLE
1.0.0 release notes

### PR DESCRIPTION
**Pull request by johnbelamaric**

### CODE REVIEW

**Comment by johnbelamaric** [comment]
@miekg @nwoods3 @chrisohaver @fturib please take a look, this is just a first draft. Natasha needs a decent draft by the end of the day 11/30.

Thanks!

File: content/blog/coredns-1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Release 1.0.0 and other recent releases have focused on improving the performance and
+functionality of the *kubernetes* plugin, since CoreDNS is now on track to eventually
+[replace kube-dns](https://github.com/kubernetes/features/issues/427) as the default
+cluster DNS in Kubernetes.
+
+As part of the [Kubernetes proposal](https://github.com/kubernetes/community/pull/1100), we have shown that CoreDNS
+not only provides more functionality than kube-dns, but performs much better while using less memory. In our tests,
+[CoreDNS](https://github.com/kubernetes/community/pull/1100#issuecomment-337747482) running against a cluster with 5000
+services was able to process 18,000 queries per second using 73MB of RAM, while 
+kube-dns](https://github.com/kubernetes/community/pull/1100#issuecomment-338329100) achieved 7,000qps using 97MB of RAM.
```
Comment (fturib) [inline content/blog/coredns-1.0.0.md]: miss left square braket "kube-dns"

File: content/blog/coredns-1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Core
+
+* Fixed a bug in the gRPC server that prevented *dnstap* from working with it.
+* Additional fuzz testing to ferret out obscure bugs
+* Documentation and configuration cleanups.
+
+## Plugins
+* *log* no longer accepts `stdout` in the configuration (use of a file was removed in a previous release). All logging is always to STDOUT. This is a backwards **incompatible** change, so be sure to check your Corefile for this.
+* *health* now checks plugins for health and reflects that in the server health.
```
Comment (miekg) [inline content/blog/coredns-1.0.0.md]: "some plugins"?

**Comment by johnbelamaric** [comment]
cc @idvoretskyi

**Comment by idvoretskyi** [comment]
@johnbelamaric looks perfect. Thank you for driving this.

**Comment by miekg** [comment]
Please also update `data/coredns.toml` with "1.0.0" release (sorry haven't found a way in Hugo to automate that).
Merging and setting this live *should* update all links.

**Comment by johnbelamaric** [comment]
Done

**Pull request merged**